sljit_src/sljitNativeRISCV_common.c

```diff
@@ -124,8 +128,27 @@ static const sljit_u8 freg_map[SLJIT_NUMBER_OF_FLOAT_REGISTERS + 3] = {
 #define ORI		(F3(0x6) | OPC(0x13))
 #define REM		(F7(0x1) | F3(0x6) | OPC(0x33))
 #define REMU		(F7(0x1) | F3(0x7) | OPC(0x33))
+#if defined __riscv_zbb
+#if defined SLJIT_CONFIG_RISCV_32
+#define REV8		(IMM_I(0x698) | F3(0x5) | OPC(0x13))
+#elif defined SLJIT_CONFIG_RISCV_64
+#define REV8		(IMM_I(0x6b8) | F3(0x5) | OPC(0x13))
+#endif
+#define ROL		(F7(0x30) | F3(0x1) | OPC(0x33))
+#define ROR		(F7(0x30) | F3(0x5) | OPC(0x33))
+#define RORI		((((sljit_ins)0x18) << 26) | F3(0x5) | OPC(0x13))
+#endif /* __riscv_zbb */
 #define SC		(F7(0xc) | OPC(0x2f))
 #define SD		(F3(0x3) | OPC(0x23))
+#if defined __riscv_zbb
+#define SEXTB		(IMM_I(0x604) | F3(0x1) | OPC(0x13))
```

Comment:
These are also defined with IMM. Perhaps we should add another F macro F5, and combine these with F7, F5. This could be used for CTZ.

Reply:
I've found F12 macro